AUTHORS.rst

```diff
@@ -1,35 +1,54 @@
 .. -*- mode: rst -*-
 
-People
+cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
+External projects used and papers to cite 
 ------
```

Comment:
Could you put as many `-` as characters in the associated title. This avoids warning 
>Title underline too short

when doc is generated